pom.xml

```diff
@@ -133,10 +135,23 @@
       <artifactId>org.eclipse.pde.core</artifactId>
       <version>${pde.core.version}</version>
     </dependency>
+	<dependency>
+      <groupId>com.atlassian.commonmark</groupId>
+      <artifactId>commonmark</artifactId>
+      <version>${commonmark.version}</version>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
+	  <plugin>
+		<groupId>org.apache.maven.plugins</groupId>
+		<artifactId>maven-surefire-plugin</artifactId>
+		<version>${org.apache.maven.plugins.version}</version>
+		<configuration>
+		  <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
```

Comment:
I'm still surprised that this file.encoding argLine should be needed, are you sure? What fails if you not add this?